static/app/components/replays/replayNewFeatureBanner.tsx

```diff
@@ -0,0 +1,116 @@
+import {CSSProperties} from 'react';
+import styled from '@emotion/styled';
+
+import newFeatureImage from 'sentry-images/spot/alerts-new-feature-banner.svg';
+
+import {IconBroadcast, IconClose} from 'sentry/icons';
+import {t} from 'sentry/locale';
+import {space} from 'sentry/styles/space';
+
+interface NewFeatureBannerProps {
```

Comment:
another nit: just call it `Props` because there is only one component in this file (`styled()` doesn't have props, doesn't count)